pycrefine.py

```
@@ -1159,6 +1163,14 @@ def _has_jump_backward(self) -> bool:
         """Return True if the code contains any backward jump."""
         return any(self._is_backward_instruction(ins) for ins in self.instructions)
 
+    def _is_null_sentinel(self, v: Any) -> bool:
+        """Check if v is a NULL sentinel (None or StackNULL). 3.11+ only."""
+        if sys.version_info < (3, 11):
+            return False
+        if v is None: return True
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)